layer/lambda_utils/lambda_utils/header_config.py

```diff
@@ -42,6 +42,11 @@ class ClientRpDetailsHeader(AbstractHeader):
 class ConnectionMetadata(AbstractHeader):
     pointer_types: list[str] = Field(alias="nrl.pointer-types")
     ods_code: str = Field(alias="nrl.ods-code")
+    ods_code_extension: str = Field(alias="nrl.ods-code-extension", default=None)
+
+    @property
+    def ods_code_parts(self):
```

Comment:
I might need someone to help explain this to me, I cant work it how this interacts myself 😓